TodoList.Web/Views/Home/Contact.cshtml

```diff
@@ -1,8 +1,4 @@
-﻿@{
-    ViewData["Title"] = "Contact";
-}
-<h2>@ViewData["Title"]</h2>
-<h3>@ViewData["Message"]</h3>
+﻿<h2>Contact</h2>
 
 <address>
     One Microsoft Way<br />
```

Comment:
Also, on the contact page, it would be nice to have a fake email address instead of the defaults. Something like: contact@todolist.com??

Reply:
Done.